pandas/_libs/groupby_helper.pxi.in

```diff
@@ -474,7 +474,8 @@ def group_min(groupby_t[:, :] out,
 def group_cummin(groupby_t[:, :] out,
                  groupby_t[:, :] values,
                  const int64_t[:] labels,
-                 bint is_datetimelike):
+                 bint is_datetimelike,
+                 int ngroups):
```

Comment:
move this arg before is_datetimelike (e.g. after labels)

can you add doc-strings for these